controllers/todoController.js

```diff
@@ -0,0 +1,101 @@
+const { Todo } = require('../models/index')
+
+class TodoController {
+    static create (req, res, next) {
+        const newData = {
+            title : req.body.title,
+            description : req.body.description,
+            status: req.body.status,
+            due_date: req.body.due_date,
+            UserId: req.currentUserId
+        }
+        Todo.create(newData)
+        .then(newData => {
+            return res.status(201).json({
+                message : "New todo-list successfully created",
+                todo: newData
+            })
+        })
+        .catch(err => {
+            return next(err)
+        })  
+    }
+
+    static findAll (req, res) {
+        let UserId = req.currentUserId
+        Todo.findAll({
+            where: {
+                UserId
+            }
+        })
+        .then(result => {
+            return res.status(200).json({
+                todo: result
+            })
+        })
+        .catch(err => {
+            return res.status(500).json({
+                name: "Internal Server",
+                errors: [{message: "Error"}]
+            })
+        })
+    }
+
+    static findOne(req, res) {
+        const id = +req.params.id
+        Todo.findByPk(id)
+        .then(result => {
+            if(result) {
+                res.status(200).json(result)
+            }
+            else {
+                res.status(404).json("Not found")
```

Comment:
dibagian ini kamu udh pernah buat jadi bisa dimasukin handler juga ntar

return res.status(404).json({
  name: "Not Found",
  errors: [{message: "Todo not found"}]
})